.github/workflows/scanner-update-NVD-data.yaml

```diff
@@ -1,9 +1,6 @@
 name: Fetch NVD Data Feeds
 
 on:
-  push:
-    branches:
-    - "yli3/**"
```

Comment:
We should still restrict which branches this should run, no? We don't need this running in every development branch, overwriting the NVD data with potential changes.

Reply:
I think this should do it:
https://github.com/stackrox/stackrox/actions/workflows/update-vulnerabilties.yaml?query=branch%3Amaster

If you select release branch for example, there's no workflow